.github/workflows/checks.yml

```diff
@@ -73,8 +73,36 @@ jobs:
       - name: Test
         run: npm run test
 
+  e2e:
+    runs-on: ubuntu-20.04
+    timeout-minutes: 60
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Cache and restore samples
+        uses: actions/cache@v2
+        id: samples
```

Comment:
What does "samples" refer to?

Reply:
Sorry for my confusion but what's in the samples folder? What's being cached?

Reply:
Answered https://github.com/dfinity/gix-components/pull/181#discussion_r1156127139